Blocks/Persistence/test.cc

```diff
@@ -294,6 +299,12 @@ TEST(PersistenceLayer, File) {
   }
 
   EXPECT_EQ(
+      "#signature\t{\"exposed_namespace\":\"namespace\",\"top_level_name\":\"top_level_name\",\"schema\":{\"types\":[["
```

Comment:
You know what? Perhaps taking the `Signature` object outside, `EXPECT_EQ`-ing its `JSON()` with a string, and then just `+`-ing that string into the expected value would be a cleaner solution?